bgpd/bgp_vty.c

```diff
@@ -1860,6 +1878,31 @@ DEFUN (no_bgp_always_compare_med,
 	return CMD_SUCCESS;
 }
 
+
+DEFUN(bgp_ebgp_requires_route_map, bgp_ebgp_requires_route_map_cmd,
+      "bgp ebgp-requires-route-map",
+      "BGP specific commands\n"
+      "Require in and out route-maps for eBGP peers (RFC8212)\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+	bgp->ebgp_requires_route_map = DEFAULT_EBGP_POLICY_ENABLED;
+	bgp_clear_all_ebgp_peers(bgp);
+	return CMD_SUCCESS;
+}
+
+DEFUN(no_bgp_ebgp_requires_route_map, no_bgp_ebgp_requires_route_map_cmd,
+      "no bgp ebgp-requires-route-map",
+      NO_STR
+      "BGP specific commands\n"
+      "Require in and out route-maps for eBGP peers (RFC8212)\n")
+{
+	VTY_DECLVAR_CONTEXT(bgp, bgp);
+	bgp->ebgp_requires_route_map = DEFAULT_EBGP_POLICY_DISABLED;
+	bgp_clear_all_ebgp_peers(bgp);
```

Comment:
As above, the hard clear doesn't make sense.  Additionally, since the effect of this change would only be to allow additional routes that were previously discarded, there is no "safety" reason to enforce any clear.  We can leave that to the operator (maybe print a message about it.)